tests/testStore.js

```diff
@@ -1,26 +0,0 @@
-import alt from '../src/alt/alt'
```

Comment:
No, they got deleted. This is the error. 
```
Warning: ReactDOMComponent: Do not access .props of a DOM node; instead, recreate the
props as `render` did originally or read the DOM properties/attributes directly from this node
(e.g., this.refs.box.className). This DOM node was rendered by `TestComponent`.
```
Trying to find out what is triggering this and the fix?